omod/src/main/resources/apps/overallActions_extension.json

```diff
@@ -29,5 +29,15 @@
         "icon": "icon-link",
         "order": 13,
         "requiredPrivilege": "Task: coreapps.mergeVisits"
+    },
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "script",
+        "label": "coreapps.task.deletePatient.label",
+        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
```

Comment:
@djazayeri @wluyima @lluismf Alright, so after @psbrandt 's help in IRC, I finally figured out what the problem is. The code written here is *essentially* correct.

After debugging this statement in the JS console in the browser, the error is at the line `delPatient.showDeletePatientCreationDialog(40a039bb-f088-46b1-b988-035fae003fb6)` which is essentially correct. There need to be double quotes around this uuid.

So I try everything ranging from  `{{patient.uuid}}`, `'{{patient.uuid}}'`, `{'{patient.uuid}'}`, `{{'patient.uuid'}}`, `patient.uuid` `\"{{patient.uuid}}\"` but the error in the console is still `Illegal token', `Statement missing`. Essentially, whoever is sending the UUID, the JS isn't recieving it in String format. When I try to send this in the form of a string in the JSON, it isn't accepting.

THAT is the problem